roles/rsyslog/defaults/main.yaml

```diff
@@ -7,6 +7,11 @@
 # Enable or disable ``rsyslog`` management.
 rsyslog_enabled: true
 
+# rsyslog_in_image
+#
+# true if the target host is a container and use rsyslog in the image.
+rsyslog_in_image: false
+
```

Comment:
Is it a user-settable parameter? I don't see it in docs.

Reply:
I think it is user-settable.  If `false` this means rsyslog is running on the host as a host package via systemd - the role needs to ensure that the rsyslog rpm packages are installed.  If `true`, this means rsyslog is running in a container - the role needs to ensure the right container is running - no rpm packages required.

Reply:
@richm, do you think this parameter should be in the doc to advertise?  Please note that the efforts we made in the openshift logging for rsyslog at the last moment are not reflected to this logging role.  Do you think they should be?

Reply:
>  Do you think they should be?

Not sure, but let's worry about that in the future.  For now, for this PR, if `rsyslog_in_image` is settable by a user, and actual does something, let's doc it.

Reply:
Will do.  Thanks.